ambari-server/src/main/python/ambari_server/dbCleanup.py

```diff
@@ -106,6 +111,9 @@ def run_db_purge(options):
     )
     return 1
 
+  properties = get_ambari_properties()
+  parse_properties_file(options)
+  options.database_index = LINUX_DBMS_KEYS_LIST.index(properties[JDBC_DATABASE_PROPERTY])
```

Comment:
Without these, it tries to execute `ensure_jdbc_driver_is_installed()` with postgresql even though ambari-server is configured with another RDBMS.